### DISCUSSION TITLE
[UX] Unexpected behaviour on pagination edges

### DISCUSSION THREAD

**Discussion by dianfishekqi** [livewire/livewire#4672] [7. Bugs (archived)]
Hello there,

On android when on first or last page clicking on pagination spans triggers google assistant, we have fixed the problem by disabling user-select on these spans using CSS.

https://user-images.githubusercontent.com/4225509/154026274-4c6f4e84-0244-4a03-b65e-78ba7e60ad17.mp4

**Comment by joshhanley** [comment]
@dianfishekqi I don't have an Android, so not sure what is going on. Are you able to explain why that triggers the google assistant?

Thanks!

**Reply by dianfishekqi** [reply]
Hello @joshhanley,
as you can see [here](https://github.com/livewire/livewire/blob/master/src/views/pagination/simple-tailwind.blade.php#L7) a span is used when there is no previous page, which google assistant treats like text, maybe we should just toggle the disabled attributes on the buttons or add `user-select none` to the spans.

Thanks

**Reply by dianfishekqi** [reply]
@joshhanley here is a PR https://github.com/livewire/livewire/pull/4680

**Reply by joshhanley** [reply]
@dianfishekqi awsome thanks! Have commented on the PR 🙂
